src/TSMapEditor/UI/Windows/TriggersWindow.cs

```diff
@@ -2342,5 +2358,13 @@ private void OpenCreateRandomTriggersSetWindow()
             createRandomTriggerSetWindow.Open();
             PutOnBackground();
         }
+        private void Map_TriggersChanged(object sender, EventArgs e)
```

Comment:
Leave a blank line between functions.

```suggestion
        }

        private void Map_TriggersChanged(object sender, EventArgs e)
```